### PR TITLE
[unifi] Use createHttpClient

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Review by jlaur — APPROVED** [review]
Thanks!

File: bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/UniFiThingHandlerFactory.java [hunk 1]
```diff
@@ -50,9 +50,7 @@ public class UniFiThingHandlerFactory extends BaseThingHandlerFactory {
 
     @Activate
     public UniFiThingHandlerFactory(@Reference final HttpClientFactory httpClientFactory) {
-        // [wip] mgb: disabled due to missing common name attributes with certs
-        // this.httpClient = httpClientFactory.getCommonHttpClient();
-        httpClient = new HttpClient(new SslContextFactory.Client(true));
+        httpClient = httpClientFactory.createHttpClient(BINDING_ID, new SslContextFactory.Client(true));
```
Comment (jlaur) [inline bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/UniFiThingHandlerFactory.java:53]: I'm wondering if we in core even could have a common client with this often used configuration?

Comment (lolodomo) [reply bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/UniFiThingHandlerFactory.java:53]: At the same time, creating such client (trust all) should be probably avoided. So providing a method for that could be a not so good idea.

Comment (jlaur) [reply bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/UniFiThingHandlerFactory.java:53]: Good point.

**Pull request merged**